src/templates/en/2020/stories/page_content.html

```diff
@@ -1897,11 +1897,6 @@
                   </div>
                 </div>
               </div>
-              <div data-amp-original-style="position:absolute;pointer-events:none;left:31.79612%;top:95.79288%;width:70.87379%;height:2.26537%;opacity:1" class="amp-wp-ccac7d2">
-                <div id="el-c9dc151c-4749-4c14-b1c8-9a9fced0d640" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0" class="amp-wp-a1b70b8">
-                  <p class="fill amp-wp-24c460b" data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:-0.6164383561643838% 0;font-family:'Lato',sans-serif;font-size:0.194175em;line-height:1.5;text-align:initial;padding:0% 0%;color:#000000"><span data-amp-original-style="color: #f7f779" class="amp-wp-66f5b53">*This chapter currently refers to data from 2019.</span></p>
-                </div>
-              </div>
```

Comment:
Nit: could remove the CSS referring to `amp-wp-ccac7d2`, `amp-wp-a1b70b8`, `amp-wp-24c460b`, and `amp-wp-66f5b53`

Reply:
Ditto for the other story with the one removed paragraph

Reply:
Yes for `amp-wp-ccac7d2` and `amp-wp-24c460b`

No for `amp-wp-a1b70b8` and `amp-wp-66f5b53` as still being used.

Reply:
And did the other one now too.